app/components/IpPoolUtilization.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright Oxide Computer Company
+ */
+
+import { parseIpUtilization, type IpPoolUtilization } from '~/api'
+import { Badge } from '~/ui/lib/Badge'
+import { BigNum } from '~/ui/lib/BigNum'
+
+const IpUtilFrac = (props: { allocated: number | bigint; capacity: number | bigint }) => (
+  <>
+    <BigNum className="text-default" num={props.allocated} /> /{' '}
+    <BigNum className="text-tertiary" num={props.capacity} />
+  </>
+)
+
+export function IpUtilCell(util: IpPoolUtilization) {
+  const { ipv4, ipv6 } = parseIpUtilization(util)
+
+  if (ipv6.capacity === 0n) {
+    return (
+      <div className="space-y-1">
+        <IpUtilFrac {...ipv4} />
+      </div>
+    )
+  }
```

Comment:
This should be what users see because they almost certainly have no IPv6 ranges.